cluster/src/dunit/scala/io/snappydata/cluster/SplitSnappyClusterDUnitTest.scala

```diff
@@ -342,8 +342,7 @@ object SplitSnappyClusterDUnitTest
       prop: Properties,
       locatorClientPort: Int): Unit = {
 
-    val snc: SnappyContext = getSnappyContextForConnector(locatorPort,
-      locatorClientPort)
+    val snc: SnappyContext = getSnappyContextForConnector(locatorClientPort)
```

Comment:
locatorPort was for locator discovery by peers which was being used earlier in split cluster mode. As we have moved to smart connector as a thin client, this was redundant.